metaworld/policies/policy.py

```diff
@@ -0,0 +1,58 @@
+import abc
+import warnings
+
+import numpy as np
+
+
+def move(from_xyz, to_xyz, p):
+    """Computes action components that help move from 1 position to another
+
+    Args:
+        from_xyz (np.ndarray): The coordinates to move from (usually current position)
+        to_xyz (np.ndarray): The coordinates to move to
+        p (float): constant to scale response
+
+    Returns:
+        (np.ndarray): Response that will decrease abs(to_xyz - from_xyz)
+
+    """
+    error = to_xyz - from_xyz
+    response = p * error
+
+    if np.any(np.absolute(response) > 1.):
+        warnings.warn('PID constants may be too high. Environments clip response to [-1, 1]')
+
+    return response
+
+
+class Policy(abc.ABC):
+    def __init__(self, p=1.):
+        self._p = p
+
+    @staticmethod
+    @abc.abstractmethod
+    def parse_obs(obs):
+        """Pulls pertinent information out of observation and places in a dict.
+
+        Args:
+            obs (np.ndarray): Observation which conforms to env.observation_space
+
+        Returns:
+            dict: Dictionary which contains information from the observation
+        """
+        raise NotImplementedError
```

Comment:
please just leave abtractmethods blank. NotImplementedError is a bandaid from Python 2